src/IniFileParser/Model/IniData.cs

```diff
@@ -90,9 +89,9 @@ public IniParserConfiguration Configuration
         {
             get
             {
-                if (_sections.ContainsSection(sectionName)) return _sections[sectionName];
+                if (!_sections.ContainsSection(sectionName)) _sections.AddSection(sectionName);
 
-                return null;
```

Comment:
Sorry, you need to put back the null return value. Returning an empty string is fine, and probably safer. But changing the return value qualifies as a breaking change, and I don't want to introduce such a change in a minor fix.